src/main/java/frame/panels/SandboxPanel.java

```diff
@@ -61,7 +69,10 @@ private void addDate()
         JPanel transactionDatePanel = new JPanel();
         transactionDatePanel.setLayout(new BoxLayout(transactionDatePanel, BoxLayout.PAGE_AXIS));
         JLabel transactionDate = new JLabel("Transaction Date");
-        transactionDateInput = new JTextField();
+        transactionDateSettings = new DatePickerSettings();
+        transactionDateInput = new DatePicker(transactionDateSettings);
```

Comment:
Just wanted to confirm: this transactionDateSettings can be passed in before its date range limits are set?

Reply:
yup; I've tested it